hardware/readme.md

```diff
@@ -0,0 +1,45 @@
+# Hardware
+
+## Ports, Usages, Pins, etc.
+
+| Port    | Usage      | Pin | Notes                    |
+|---------|------------|-----|--------------------------|
+| NRST    | GND        |  1  |                          |
+| PA1     | SenseLED1+ |  2  |                          |
+| PA2     | GLED3      |  3  |                          |
+| VSS     | GND        |  4  |                          |
+| PD0     | GLED2      |  5  |                          |
+| VDD     | VCC        |  6  |                          |
+| PC0     | GLED1      |  7  |                          |
+| SDA     | SDA        |  8  | PC1                      |
+| SCL     | SCL        |  9  | PC2                      |
+| PC3     | Button1    | 10  |                          |
+| PC4     | Button2    | 11  |                          |
+|         | N/C        | 12  |                          |
```

Comment:
Pin 12, PC5, is connected to GPIO2 on the SAO header.

Reply:
oh, I think the MCU pinout circuit image I have must be old.